FlashDevelop/Managers/ScintillaManager.cs

```diff
@@ -253,7 +253,8 @@ public static void ApplySciSettings(ScintillaControl sci, Boolean hardUpdate)
                 sci.IsTabIndents = settings.TabIndents;
                 sci.IsUseTabs = settings.UseTabs;
                 sci.IsViewEOL = settings.ViewEOL;
-                sci.ScrollWidth = settings.ScrollWidth;
+                sci.ScrollWidth = Math.Max(settings.ScrollWidth, 1);
```

Comment:
Yep, setting it to 0 sets it to the scintilla default (which is I think either 2000 or 5000).